docs/cloud/introduction.md

```diff
@@ -5,77 +5,77 @@ order: 1
+To finalize the sign-up process, you must confirm your email address by clicking the link in the confirmation email. It might happen that the confirmation email doesn't get through, please check the junk email folder in your email client as you might find it there.
 
-### Login to Cloud console
+### Login to the Event Store Cloud console
 
 With the new account, you can log in to the [Cloud console](https://console.eventstore.cloud). 
 
-In the console, you first get to the list of organisations that you have access to.
+In the console, you first get to the list of organizations you have access to.
 
 ![Cloud organisations](./images/intro/cloud-console-orgs.png)
 
-You start with an empty list, so you'll need to add an organisation.
+You start with an empty list, so you must add an organization.
 
 ::: tip
-The Cloud console has the concept of Context. You'd normally be in the organisation context or in the project context.
+The Cloud console has the concept of Context. You'll typically be in the organization or project context.
 :::
 
 ### Organisations
 
-With your account, you might have more than one organisation. Each organisation has its own billing account and therefore is invoiced separately for all the resources within the organisation. For example, if you have several customers and want to have a separate EventStoreDB cluster for each customer and also be able to bill the customer for the cloud resources, you can separate each customer into its own organisation.
+With your account, you might have more than one organization. Each organization has its own billing account and is, therefore, invoiced separately for all the resources within the organization. For example, if you have several customers and want to have a separate EventStoreDB cluster for each customer and also be able to bill the customer for the cloud resources, you can separate each customer into its own organization.
 
-When you click on an organisation in the list, you get to the projects screen, where you can see all the projects for the selected organisation.
+When you click on an organization in the list, you get to the projects screen, where you can see all the projects for the selected organization.
 
 ![Projects within the organisation](./images/intro/cloud-org-projects.png)
 
-Within the scope of the organisation, you also have the list of users and roles, billing information, alerts, and so on.
+Within the organization's scope, you also have the list of users and roles, billing information, alerts, etc.
 
-You can always switch to another organisation by clicking on the selected organisation name and either selecting the organisation from the list or clicking on `All organizations` to return to the list.
+You can always switch to another organization by clicking on the selected organization name, either selecting it from the list or clicking on `All organizations` to return to the list.
```

Comment:
If you change it to `z`, I guess the header above also needs to be updated?

Reply:
done